tools/mkdoc.py

```diff
@@ -241,12 +251,18 @@ def run(self):
             parameters.append('-isysroot')
             parameters.append(sysroot_dir)
 
+    std = '-std=c++11'
+
     for item in sys.argv[1:]:
+        if item.startswith('-std='):
+            std = item
         if item.startswith('-'):
             parameters.append(item)
         else:
             filenames.append(item)
 
+    parameters.append(std)
```

Comment:
Would this add `std=*` given as an arg twice?

Reply:
It would. Line 259 needs to be an `elif`, I think.